src/Vegetation/PlantHydraulics.jl

```diff
@@ -431,28 +441,29 @@ function flux_out_roots(
     p::ClimaCore.Fields.FieldVector,
     t::FT,
 )::FT where {FT}
-    @unpack vg_α, vg_n, vg_m, ν, S_s, K_sat = model.parameters
-
+    @unpack vg_α, vg_n, vg_m, ν, S_s, K_sat, root_distribution = model.parameters
     S_l_stem = effective_saturation(ν, Y.vegetation.ϑ_l[1])
-
-    #How to ungeneralize
-
-    p_stem = water_retention_curve.(vg_α, vg_n, vg_m, S_l_stem, ν, S_s)
-    return sum(
+    ψ_stem = water_retention_curve.(vg_α, vg_n, vg_m, S_l_stem, ν, S_s)
+    sum_flux_out_roots = sum(
         flux.(
             model.domain.root_depths,
             model.domain.compartment_midpoints[1],
-            re.p_soil(t),
-            p_stem,
+            re.ψ_soil(t),
+            ψ_stem,
             vg_α,
             vg_n,
             vg_m,
             ν,
             S_s,
-            K_sat[:root],
-            K_sat[model.domain.compartment_labels[1]],
-        ),
-    )
+            K_sat[:root].(model.domain.root_depths),
```

Comment:
My question about K_sat[:root] was why it is a function now, instead of a scalar? Is this something we actually have data to constrain?

Reply:
Was changed following a convo we had had about converting between conductance and conductivity, but the change isn't necessary in the end as nothing to constrain with indeed.